browser/app/profile/firefox.js

```diff
@@ -249,7 +249,8 @@ pref("browser.defaultbrowser.notificationbar.checklimit", 10000);
 // 0 = blank, 1 = home (browser.startup.homepage), 2 = last visited page, 3 = resume previous browser session
 // The behavior of option 3 is detailed at: http://wiki.mozilla.org/Session_Restore
 pref("browser.startup.page",                1);
-pref("browser.startup.homepage",            "http://replay.io/view");
+// [Replay] - Override the default homepage to be the replay library
+pref("browser.startup.homepage",            "http://app.replay.io/");
```

Comment:
Seems like this should be `https` actually

Reply:
Good call!